frontend/src/components/TransactionItem.js

```diff
@@ -1,20 +1,28 @@
 import React from "react";
+import { Container, Row, Col, Button } from "react-bootstrap";
+import { FaRegTrashAlt } from "react-icons/fa";
 
 const TransactionItem = (props) => {
   return (
-    <div>
-      <div>
-        <div>{props.userexpense.title}</div>
-        <div>{props.userexpense.amount}</div>
-        <button
-          onClick={() => {
-            props.delExpense(props.userexpense.id, props.userexpense.amount);
-          }}
-        >
-          x
-        </button>
-      </div>
-    </div>
+    <Container className="px-1 pt-3 transactionContainer ">
+      <Row className="pb-1 ">
+        <Col xs={7}>
+          <p>{props.userexpense.title}</p>
+        </Col>
+        <Col xs={3}>
+          <p>{props.userexpense.amount} €</p>
+        </Col>
+        <Col xs={2}>
```

Comment:
I modified the bootstrap utility classes used here a bit to fix the center alignment and move all margin/padding related code here - please check the added commit if it looks ok still.